website/content/docs/platform/k8s/vso/sources/vault/client-cache.mdx

```diff
@@ -0,0 +1,266 @@
+  capabilities = ["create", "update"]
+}
+path "transit/decrypt/vso-client-cache" {
+  capabilities = ["create", "update"]
+}
+EOH
+```
+
+</CodeBlockConfig>
+<CodeBlockConfig>
+
+```hcl
+resource "vault_mount" "transit" {
+  namespace   = "<VAULT_NAMESPACE>"
+  path        = "transit"
+  type        = "transit"
+  description = "VSO Client Cache"
+}
+
+resource "vault_transit_secret_cache_config" "cache" {
+  namespace = vault_mount.transit.namespace
+  backend   = vault_mount.transit.path
+  size      = 500
+}
+
+resource "vault_transit_secret_backend_key" "cache" {
+  namespace        = vault_mount.transit.namespace
+  backend          = vault_mount.transit.path
+  name             = "vso-client-cache"
+  deletion_allowed = true
+}
+
+resource "vault_policy" "operator" {
+  namespace = vault_transit_secret_backend_key.cache.namespace
+  name      = "operator"
+  policy    = <<EOT
+path "${vault_mount.transit.path}/encrypt/${vault_transit_secret_backend_key.cache.name}" {
+  capabilities = ["create", "update"]
+}
+path "${vault_mount.transit.path}/decrypt/${vault_transit_secret_backend_key.cache.name}" {
+  capabilities = ["create", "update"]
+}
+EOT
+}
+```
+
+</CodeBlockConfig>
+</CodeTabs>
+
+## Step 2: Configure kubernetes auth
+
+Use the following commands/terraform to enable the kubernetes auth method and create a role for the operator to use transit with the policy from the previous step.
+
+<CodeTabs>
+<CodeBlockConfig>
+
+```shell-session
+export VAULT_NAMESPACE=<VAULT_NAMESPACE>
+
+vault auth enable kubernetes
+
+vault write auth/kubernetes/config kubernetes_host=https://kubernetes.default.svc
+
+vault write auth/kubernetes/role/operator \
+  bound_service_account_names=vault-secrets-operator-controller-manager \
+  bound_service_account_namespaces=vault-secrets-operator \
+  token_period="24h" \
+  token_policies=operator \
+  audience="vault"
+```
+
+</CodeBlockConfig>
+<CodeBlockConfig>
+
+```hcl
+ resource "vault_auth_backend" "kubernetes" {
+  namespace = "<VAULT_NAMESPACE>"
+  path      = "kubernetes"
+  type      = "kubernetes"
+}
+
+resource "vault_kubernetes_auth_backend_config" "local" {
+  namespace       = vault_auth_backend.kubernetes.namespace
+  backend         = vault_auth_backend.kubernetes.path
+  kubernetes_host = "https://kubernetes.default.svc"
+}
+
+resource "vault_kubernetes_auth_backend_role" "operator" {
+  namespace                        = vault_auth_backend.kubernetes.namespace
+  backend                          = vault_kubernetes_auth_backend_config.local.backend
+  role_name                        = "operator"
+  bound_service_account_names      = ["vault-secrets-operator-controller-manager"]
+  bound_service_account_namespaces = ["vault-secrets-operator"]
+  token_period                     = 120
+  token_policies = [
+    vault_policy.operator.name,
+  ]
+  audience = "vault"
+}
+```
+
+</CodeBlockConfig>
+</CodeTabs>
+
+## Step 3: (Optional) Add a VaultConnection
```

Comment:
I can see what you're getting at with optional, but I think this would be clearer phrased slightly differently:

```suggestion
## Step 3: Configure a VaultConnection
```

Reply:
Sure, that sounds better